middlewares/auth.js

```diff
@@ -10,6 +10,18 @@ module.exports = (req, res, next) => {
         data: "Unauthorized",
       });
     }
+    const token =
+      req.headers.authorization && req.headers.authorization.split(" ")[1];
+    console.log(token);
+    if (!token) {
```

Comment:
```suggestion
    if (!user.token) {
```
To wystarczy, bo wiemy że token jest poprawny (passport to sprawdził) a teraz sprawdzamy tylko czy user ma token

Reply:
update zrobiony